sr5824-msr9732-sm9453/project.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Interoperability in the Heterogeneous Cloud Environment"
+author:
+  - Sriram Ramesh <sriram.ramesh@nyu.edu>
+  - Metarya Ruparel <msr9732@nyu.edu>
+  - Safwan Mahmood <sm9453@nyu.edu>
+---
+
+For our Big Data & Machine Learning final project we plan to hypothesize a solution
+for **Interoperability in the Heterogeneous Cloud Environment**, a blatant issue 
+industry wide.
+
+# Motivation
+
+After the burst of cloud platforms in 2006 with the introduction of Amazon
+Web Services, a lot of companies started migrating their services to the 
+cloud platform from a bare metal lying in one of their own datacenters.
+Big hulking datacenters are a necessary evil, a barely concealed secret. 
+No one wants to build one. They’re certainly impressive – they resemble the 
+set of a science fiction movie, row upon row of silent machines. But they’re 
+wildly expensive, and their maintenance challenges are never ending. 
+
+Cloud Platforms provided these companies the ability to access shared, online computing 
+resources, without the over head of maintaining them. However, these providers often 
+offer their own proprietary applications, interfaces, APIs and infrastructures, which
+somewhat makes these providers unique. Due to the uniqueness (be it performance or costing)
+companies would want to onboard different set of services onto various providers.
+For example, Salesforce.com, which  started the wave of SaaS back in the late 1900's 
+, has only recently started it's migration from 1st party datacenters to these 3rd party
+providers. Some of their micro-services are being on-boarded to GCP, while a couple of
+others to AWS, resulting in a heterogeneous cloud environment. This heterogeneous 
+environment makes it difficult for companies to change cloud service providers; in 
+any case. Exploring capabilities to support the automated migration from one provider 
```

Comment:
Not quite what sure what the "; in any case" is referring to here.

Reply:
What we were trying to convey here was, there could be any reason for a company/individual to decide to move their service from one cloud provider to another. It could be due to the changes in the costing, or maybe the entire cloud service was disrupted (or their entire datacenter was burnt, not likely). So, what we meant to say is it is not straightforward to move services if any of these circumstances were to occur. When I re-read it, I see why it is confusing because it does not convey what we were thinking. We will make changes to reflect our thought process.